src/operator/numpy/np_roll_op.cc

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
This op seems to belong to `np_matrix_op`, move it there instead of opening a new file.